.github/scripts/generate_ci_workflows.py

```diff
@@ -76,7 +77,8 @@ def PyTorchLinuxWorkflow(
         "enable_nogpu_no_avx2_test": enable_nogpu_no_avx2_test,
         "enable_slow_test": enable_slow_test,
         "num_test_shards": num_test_shards,
-        "exclude_test": exclude_test,
+        "is_libtorch": is_libtorch,
+        "exclude_test": is_libtorch or exclude_test,   # libtorch is build only
```

Comment:
hmm I guess this is fine; I was about to say that this would be the only place where we don't just pass the parameter straight through to a field of the same name, but then I realized that #60215 already did that [in `PyTorchWindowsWorkflow`](https://github.com/pytorch/pytorch/blob/a26a9f8b75f4e8c9c7c34907a099b3a1d41f8a2c/.github/scripts/generate_ci_workflows.py#L40)